package.json

```diff
@@ -46,12 +54,15 @@
     "serialize-javascript": "6.0.2",
     "socket.io": "4.6.2",
     "socket.io-parser": "4.2.3",
-    "styled-components": "^5.3.6",
+    "styled-components": "5.3.7",
+    "tough-cookie": "4.1.3",
     "tweetnacl": "1.0.3",
-    "uglify-js": "2.4.24",
+    "ua-parser-js": "1.0.33",
+    "uglify-js": "3.14.3",
```

Comment:
You sure there are no breaking changes here?

Reply:
I have checked and did not find any issues in the application. After updating the packages, I will submit the app for UAT testing.